### PR TITLE
preprocess: load recursively and convert formats

### LINKED ISSUE #2: train loss didn't change after 15k epoch
Hi and thanks for this, it looks super interesting and I can't wait to try it out :)
I'm trying to train a model, but after 15k epochs the loss didn't go down a tiny bit (train loss ~2.008, val loss ~1.97) .. is it expected or something is not right? the rave model I'm using is trained with a v2 config, the dataset is around 10 hours of records from a single artist, although quite  heterogeneous.

### PR DESCRIPTION
**Pull request by elgiano**
In preprocess.py, search for audio files recursively, and use ffmpeg to convert them. The ffmpeg part was copied from rave, minus the multiprocessing part.
Adds an argument --extensions, to specify which extensions are searched for, with the same default selection as rave.

Fixes #2

### CODE REVIEW

**Comment by moiseshorta** [comment]
Hey!

This sounds great, I'll give it a test and merge in the coming days.

Best,
M

**Comment by elgiano** [comment]
I found a problem: when reading audiofiles I wasn't converting ints to floats properly. Sorry! I fixed it in the second commit
